scripts/categorizer.py

```diff
@@ -11,6 +15,25 @@ def find_empty_lines(line):
 
     return 0
 
+def determine_if_conditional(line_number, multiline_number, line, categorizations):
```

Comment:
You strip these lines so often coming into these funtions, why not strip the lines before doing everything...

Reply:
Well, won't that ruin the categorizations that have to do with whitespace???